checklists/engineeringFacilitator.json

```diff
@@ -1,82 +1,76 @@
 {"checklistName": "Engineering Facilitator of New Hire",
 "checklistDescription": "TODO Items for an engineering facilitator of a new hire.",
-"items": 
-	{"dayZero": { 
-		"displayName": "Start Date", 
+"items":
+	{"dayZero": {
+		"displayName": "Start Date",
 		"description": "First day on the job.",
-		"daysToComplete": 0, 
+		"daysToComplete": 0,
 		"dependsOn": []
 	},
-	"intro": { 
-		"displayName": "Introductory check in", 
+	"intro": {
+		"displayName": "Introductory check in",
 		"description": "Have an introductory check in with them",
-		"daysToComplete": 5, 
+		"daysToComplete": 5,
 		"dependsOn": ["dayZero"]
 	},
-	"addToCalendar": { 
-		"displayName": "Add to calendar", 
-		"description": "Add new hire to engineering calendar",
-		"daysToComplete": 5, 
-		"dependsOn": ["dayZero"]
-	},
-	"addToDevList": { 
-		"displayName": "Add to Dev List", 
+	"addToDevList": {
+		"displayName": "Add to Dev List",
 		"description": "Add new hire to <a href=\"https://groups.google.com/a/gsa.gov/forum/#!managemembers/18fdev/add\"> 18F developers list</a>. Also look through <a href=\"https://groups.google.com/a/gsa.gov/forum/#!forum/18fdev\">recent emails </a> to see if there are any updates they need.",
-		"daysToComplete": 5, 
+		"daysToComplete": 5,
 		"dependsOn": ["dayZero"]
 	},
-	"devIntro": { 
-		"displayName": "Dev Introduction", 
+	"devIntro": {
+		"displayName": "Dev Introduction",
 		"description": "Intro them to the #dev team via Slack",
-		"daysToComplete": 5, 
+		"daysToComplete": 5,
 		"dependsOn": ["dayZero"]
 	},
-	"groupIntro": { 
-		"displayName": "Group Intro", 
+	"groupIntro": {
+		"displayName": "Group Intro",
 		"description": "Intro them to their small engineering group via Slack",
-		"daysToComplete": 5, 
+		"daysToComplete": 5,
 		"dependsOn": ["dayZero"]
 	},
-	"facilitatorInvite": { 
-		"displayName": "Facilitator Group Invite", 
+	"facilitatorInvite": {
+		"displayName": "Facilitator Group Invite",
 		"description": "Invite them to the facilitator group meeting",
-		"daysToComplete": 5, 
+		"daysToComplete": 5,
 		"dependsOn": ["dayZero"]
 	},
-	"setGoals": { 
-		"displayName": "Set Goals & OKRs", 
+	"setGoals": {
+		"displayName": "Set Goals & OKRs",
 		"description": "Set a meeting to create goals and OKRs",
-		"daysToComplete": 30, 
+		"daysToComplete": 30,
+		"dependsOn": ["dayZero"]
+	},
```

Comment:
We/I haven't been doing this for individuals recently... @hollyallen is this still relevant?